src/styles/index.ts

```diff
@@ -3674,6 +3674,22 @@ const styles = (theme: ThemeColors) =>
             ...wordBreak.breakWord,
         },
 
+        quickActionTooltipWrapper: {
+            backgroundColor: colors.green100,
+        },
+
+        quickActionTooltipTitle: {
+            fontFamily: FontUtils.fontFamily.platform.EXP_NEUE_BOLD,
+            fontWeight: FontUtils.fontWeight.bold,
+            fontSize: variables.fontSizeNormal,
+            lineHeight: variables.lineHeightXLarge,
```

Comment:
Why do we need a huge line height here? I would think we could just vertically center the text via flex or use inner padding instead? I don't feel strongly though. Curious what we are using in the Figma.